shaders/IronChunk.shader

```diff
@@ -1,7 +1,8 @@
 shader_type spatial;
 render_mode depth_draw_alpha_prepass;
 
-uniform sampler2D texture : hint_albedo;
+uniform sampler2D Texture : hint_albedo;
+uniform sampler2D Normals;
```

Comment:
these don't follow the naming convention in other shaders

Reply:
will change it right now

Reply:
hold on, godot reimported something messed up